runtime.txt

```diff
@@ -1 +1 @@
-python-3.9.2
\ No newline at end of file
+python-3.9.2
```

Comment:
This could be upgraded to the latest 3.10 version

Reply:
This entire port is meant to target Python 3.9. Porting to 3.10 can be left for another less substantial PR.